<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelecaoMesAno</class>
 <widget class="QDialog" name="SelecaoMesAno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleção Mes, Ano</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Entre com Mês e Ano</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mês:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMes">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ano:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinAno">
          <property name="minimum">
           <number>2000</number>
          </property>
          <property name="maximum">
           <number>2100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancelar">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
